templates/state_machine.yaml

```diff
@@ -726,7 +726,9 @@ Resources:
     Type: AWS::SQS::Queue
     Properties:
       KmsMasterKeyId: alias/aws/sqs
-      VisibilityTimeout: 1800
+      VisibilityTimeout: 43200
+      ContentBasedDeduplication: true
+      FifoQueue: true
```

Comment:
Thanks, you're right. I updated the names so that it shouldn't be longer than the one we already have in the DelStack. I think it's not very elegant but still better than bringing the Queue to the parent stack 😢

Reply:
OK cool. It may be worth understanding what the stack name limit is because if it's relatively small e.g. <10 chars, then we should document that limitation?